pkg/graph/resolvers/trb_request.go

```diff
@@ -438,18 +438,3 @@ func GetTRBRequesterInfo(ctx context.Context, requesterEUA string) (*models.User
 
 	return requesterInfo, nil
 }
-
-// GetTRBUserComponent retrieves the component of a TRB user from the TRB attendees table
-func GetTRBUserComponent(ctx context.Context, store *storage.Store, euaID *string, trbRequestID uuid.UUID) (*string, error) {
```

Comment:
There's a `resolvers/trb_request_attendee.go` file that I thought was more appropriate for this